### PR TITLE
Makes the SAD need synthflesh for monkeys

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

If you're put in the manual transmission, you will learn the manual transmission. Makes it so you have to use synthflesh to SAD monkeys (typically brain transplants). 
You can't circumvent husks anymore, you need that synthflesh or you will be in a monkey body. 

## Why It's Good For The Game
Makes circumventing medical by doing improvised cloning have a cost other than free. 

## Changelog

:cl:
balance: synthflesh is required for monkey conversion
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 1]
```diff
@@ -63,13 +63,17 @@
...
 
 /obj/machinery/self_actualization_device/close_machine(atom/movable/target, density_to_set = TRUE)
 	..()
 	playsound(src, 'sound/machines/click.ogg', 50)
 	if(!occupant)
 		return FALSE
-	if(!ishuman(occupant))
+	if(!ishuman(occupant) || istype(occupant, /mob/living/carbon/human/species/monkey))// BUBBER EDIT BEGIN
+		say("Warning: Occupant lacks actualizing genes. Please insert synthflesh or select new user. ")
+		playsound(loc, 'sound/machines/chime.ogg', 30, FALSE) // BUBBER EDIT END
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:76]: This check is failing in all cases when a mob tries to use the SAD. The SAD is broken.

**Review by Majkl-J — CHANGES_REQUESTED** [review]
The amount needed should probably be the same as unhusking not 300u

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 2]
```diff
@@ -63,13 +63,17 @@
 /obj/machinery/self_actualization_device/Initialize(mapload)
 	. = ..()
 	update_appearance()
+	create_reagents(300, OPENCONTAINER) // BUBBER EDIT
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:66]:
```suggestion
	create_reagents(600, OPENCONTAINER) // BUBBER EDIT
```
No reason to make it this restricted especially in high units of synthflesh

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 3]
```diff
@@ -84,6 +88,10 @@
 	. = ..()
 	if(!powered() || !occupant || state_open)
 		return FALSE
+	if(!reagents.has_reagent(/datum/reagent/medicine/c2/synthflesh, 300)) // BUBBER EDIT BEGIN
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:91]:
```suggestion
	if(!reagents.has_reagent(/datum/reagent/medicine/c2/synthflesh, 200)) // BUBBER EDIT BEGIN
```
I agree with waterpig.

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm [hunk 4]
```diff
@@ -152,7 +160,9 @@
 	else if (tgui_alert(occupant, "The SAD you are within is about to rejuvenate you, resetting your body to its default state (in character preferences). Do you consent?", "Rejuvenate", list("Yes", "No"), timeout = 10 SECONDS) != "Yes")
 		failure = TRUE // defaults to rejecting it unless specified otherwise
 		failure_text = "ERROR: Occupant genes have willfully rejected the procedure. You may try again if you think this was an error."
-
+	if(!reagents.remove_reagent(/datum/reagent/medicine/c2/synthflesh, 300)) // BUBBER EDIT BEGIN
```
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:163]:
```suggestion
	if(!reagents.remove_reagent(/datum/reagent/medicine/c2/synthflesh, 200)) // BUBBER EDIT BEGIN
```
Per above

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm (hunk 3 above)
Comment (Majkl-J) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:91]: Use SYNTHFLESH_LING_UNHUSK_AMOUNT

File: modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm (hunk 2 above)
Comment (Cyprex) [reply modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm:66]: I imagine you can't withdraw this?

**Comment by Cyprex** [comment]
300u is too much and this is unfortunately an integral feature of our server- 

Actually hold on, can't you just use genetics to turn monkeys into humans to circumvent this? If so it sounds like this PR achieves nothing.

**Comment by nevimer** [comment]
> 300u is too much and this is unfortunately an integral feature of our server-
> 
> Actually hold on, can't you just use genetics to turn monkeys into humans to circumvent this? If so it sounds like this PR achieves nothing.

No, the point is that circumventing medical by just doing brain transplants onto a monkey-human and tossing a player into the sad immediately after defib for a free aheal and pref apply is no longer a thing. We have an issue where players see patients with more than one wound or problem coming in and just deciding that its better to brain swap than to fix rather minor problems.

**Comment by Majkl-J** [comment]
Stale. Reopen when work continues

**Pull request closed without merging**